=== PROXIES FUNCIONANDO ===
SOCKS5 99.79.124.70:80

=== PROXIES FALLIDOS ===
SOCKS5 37.120.172.84:80 - Razón: HTTPConnectionPool(host='httpbin.org', port=80): Max retries exceeded with url: /ip (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000022D7529A840>: Failed to establish a new connection: Socket error: SOCKS5 proxy server sent invalid data'))
SOCKS5 51.75.206.209:80 - Razón: HTTPConnectionPool(host='httpbin.org', port=80): Max retries exceeded with url: /ip (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000022D7529B0E0>: Failed to establish a new connection: Error connecting to SOCKS5 proxy 51.75.206.209:80: timed out'))
SOCKS5 50.223.246.237:80 - Razón: HTTPConnectionPool(host='httpbin.org', port=80): Max retries exceeded with url: /ip (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000022D7529B8C0>: Failed to establish a new connection: Error connecting to SOCKS5 proxy 50.223.246.237:80: timed out'))
SOCKS5 50.174.7.159:80 - Razón: HTTPConnectionPool(host='httpbin.org', port=80): Max retries exceeded with url: /ip (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000022D7529B5C0>: Failed to establish a new connection: Error connecting to SOCKS5 proxy 50.174.7.159:80: timed out'))
SOCKS5 50.207.199.87:80 - Razón: HTTPConnectionPool(host='httpbin.org', port=80): Max retries exceeded with url: /ip (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000022D7529AEA0>: Failed to establish a new connection: Error connecting to SOCKS5 proxy 50.207.199.87:80: timed out'))
SOCKS5 32.223.6.94:80 - Razón: HTTPConnectionPool(host='httpbin.org', port=80): Max retries exceeded with url: /ip (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000022D7529A8A0>: Failed to establish a new connection: Error connecting to SOCKS5 proxy 32.223.6.94:80: timed out'))
SOCKS5 89.58.53.205:80 - Razón: HTTPConnectionPool(host='httpbin.org', port=80): Max retries exceeded with url: /ip (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000022D7529BEF0>: Failed to establish a new connection: Socket error: SOCKS5 proxy server sent invalid data'))
SOCKS5 50.207.199.80:80 - Razón: HTTPConnectionPool(host='httpbin.org', port=80): Max retries exceeded with url: /ip (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000022D752D4920>: Failed to establish a new connection: Error connecting to SOCKS5 proxy 50.207.199.80:80: timed out'))
SOCKS5 50.207.199.83:80 - Razón: HTTPConnectionPool(host='httpbin.org', port=80): Max retries exceeded with url: /ip (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000022D732FF980>: Failed to establish a new connection: Error connecting to SOCKS5 proxy 50.207.199.83:80: timed out'))
SOCKS5 50.174.7.153:80 - Razón: HTTPConnectionPool(host='httpbin.org', port=80): Max retries exceeded with url: /ip (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000022D75298560>: Failed to establish a new connection: Error connecting to SOCKS5 proxy 50.174.7.153:80: timed out'))
SOCKS5 50.202.75.26:80 - Razón: HTTPConnectionPool(host='httpbin.org', port=80): Max retries exceeded with url: /ip (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000022D7529B950>: Failed to establish a new connection: Error connecting to SOCKS5 proxy 50.202.75.26:80: timed out'))
SOCKS5 50.239.72.18:80 - Razón: HTTPConnectionPool(host='httpbin.org', port=80): Max retries exceeded with url: /ip (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000022D7529B500>: Failed to establish a new connection: Error connecting to SOCKS5 proxy 50.239.72.18:80: timed out'))
SOCKS5 50.175.212.66:80 - Razón: HTTPConnectionPool(host='httpbin.org', port=80): Max retries exceeded with url: /ip (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000022D752D4080>: Failed to establish a new connection: Error connecting to SOCKS5 proxy 50.175.212.66:80: timed out'))
SOCKS5 50.217.226.47:80 - Razón: HTTPConnectionPool(host='httpbin.org', port=80): Max retries exceeded with url: /ip (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000022D752D57C0>: Failed to establish a new connection: Error connecting to SOCKS5 proxy 50.217.226.47:80: timed out'))
SOCKS5 50.239.72.16:80 - Razón: HTTPConnectionPool(host='httpbin.org', port=80): Max retries exceeded with url: /ip (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000022D7529B740>: Failed to establish a new connection: Error connecting to SOCKS5 proxy 50.239.72.16:80: timed out'))
SOCKS5 50.239.72.19:80 - Razón: HTTPConnectionPool(host='httpbin.org', port=80): Max retries exceeded with url: /ip (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000022D7529BE00>: Failed to establish a new connection: Error connecting to SOCKS5 proxy 50.239.72.19:80: timed out'))
SOCKS5 50.217.226.40:80 - Razón: HTTPConnectionPool(host='httpbin.org', port=80): Max retries exceeded with url: /ip (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000022D750296A0>: Failed to establish a new connection: Error connecting to SOCKS5 proxy 50.217.226.40:80: timed out'))
SOCKS5 50.221.74.130:80 - Razón: HTTPConnectionPool(host='httpbin.org', port=80): Max retries exceeded with url: /ip (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000022D752D43E0>: Failed to establish a new connection: Error connecting to SOCKS5 proxy 50.221.74.130:80: timed out'))
